### ISSUE TITLE
wrong Content-Type for web UI 

### ISSUE THREAD

**Issue by Wing924** [Comcast/trickster#209]
```
$ curl -vs localhost:8000
* Rebuilt URL to: localhost:8000/
*   Trying ::1...
* TCP_NODELAY set
* Connected to localhost (::1) port 8000 (#0)
> GET / HTTP/1.1
> Host: localhost:8000
> User-Agent: curl/7.54.0
> Accept: */*
>
< HTTP/1.1 200 OK
< Access-Control-Allow-Origin: *
< Content-Type: application/json
< Date: Fri, 19 Apr 2019 03:50:59 GMT
< Transfer-Encoding: chunked
<
<!DOCTYPE html>
<html lang="en">
  <head>
    <meta http-equiv="Content-Type" content="text/html; charset=utf-8">
    <title>Prometheus Time Series Collection and Processing Server</title>
    <link rel="shortcut icon" href="/static/img/favicon.ico?v=">
    <script src="/static/vendor/js/jquery.min.js?v="></script>
    <script src="/static/vendor/bootstrap-3.3.1/js/bootstrap.min.js?v="></script>

    <link type="text/css" rel="stylesheet" href="/static/vendor/bootstrap-3.3.1/css/bootstrap.min.css?v=">
...
```

Since response is html, content type should be `text/html` or missing.

version: 0.1.9

**Comment by jranson** [comment]
@Wing924 Thanks for reporting this. It is already fixed in the 1.0 codebase on the `next` branch, which properly returns _all_ headers back from the origin.

Separately, making a request to `http://prometheus:9090/` should issue a 302 Redirect to `/graph`. However, I see in your curl output above that you received a 200 response. I looked into this and found that Trickster is following redirects automatically and returning the final response. It turns out that this is due to the default behavior of golang's net/http client implementation. In the 1.0 release, I will submit a PR to disable the automatic following, as Trickster is a proxy and should return, not follow, redirects.

**Comment by jacksontj** [comment]
This bug was introduced in #196 where the promFullProxyHandler was changed to unconditionally set the response type to application/json.

So we have 2 fixes (1) revert #196 or (2) change `writeResponse` to proxy the content-type from the origin -- preferences @jranson ?

**Comment by jranson** [comment]
@jacksontj Thanks for digging into all of that. For 0.1.x, option 2 is definitely the right way to go. I will take a look later this weekend and get a PR ready unless someone can beat me to it!
